<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedNamingDialog</class>
 <widget class="QDialog" name="AdvancedNamingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced capture naming</string>
  </property>
  <widget class="QGroupBox" name="advancedNamingGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Advanced capture naming options:</string>
   </property>
   <widget class="QRadioButton" name="formatNtscRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>NTSC</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">formatButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="formatPalRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>PAL</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">formatButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="typeCavRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CAV</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">discTypeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="typeClvRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CLV</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">discTypeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="discTitleLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="discTitleCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Disc title:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="discTypeCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Disc type:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="formatCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Format:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="discSideCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Disc side:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="notesLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="notesCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="audioCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="audioDefaultRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">audioButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="audioAnalogueRadioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Analogue only</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">audioButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="audioAc3RadioButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AC3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">audioButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="audioDtsRadioButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DTS</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">audioButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QSpinBox" name="discSideSpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="mintCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Mint marks:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mintLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="durationCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Append capture duration</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>discTitleCheckBox</tabstop>
  <tabstop>discTitleLineEdit</tabstop>
  <tabstop>discTypeCheckBox</tabstop>
  <tabstop>typeCavRadioButton</tabstop>
  <tabstop>typeClvRadioButton</tabstop>
  <tabstop>formatCheckBox</tabstop>
  <tabstop>formatNtscRadioButton</tabstop>
  <tabstop>formatPalRadioButton</tabstop>
  <tabstop>discSideCheckBox</tabstop>
  <tabstop>notesCheckBox</tabstop>
  <tabstop>notesLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="formatButtonGroup"/>
  <buttongroup name="discTypeButtonGroup"/>
  <buttongroup name="audioButtonGroup"/>
 </buttongroups>
</ui>
